core/arch/arm/plat-sunxi/psci.c

```diff
@@ -0,0 +1,211 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright (c) 2018, Linaro Limited
```

Comment:
I seem to recognize some of the stuff below so I guess that the content of this file has some history.